lib/parser.rb

```diff
@@ -29,7 +29,7 @@ def command_type
   end
 
   def arg1
-    if command_type == :C_PUSH
+    if command_type == :C_PUSH || command_type == :C_POP
```

Comment:
Now that we’re comparing a single variable against multiple values, another option is something like:

```suggestion
    if [:C_PUSH, :C_POP].include?(command_type)
```

Or we could turn it into a `case command_type` and say `when :C_PUSH, :C_POP`. So many choices!